docs/adguard-vpn-for-mac/overview.md

```diff
@@ -40,6 +40,15 @@ Also you can not only add websites to Exclusions but also choose among the lists
 
 Exclusion list can be easily configured. If you've added a domain and some of its subdomains, they'll be grouped inside the root one. When adding a root domain (`example.com`) its mask is added too (`*.example.com`).
 
+:::info Browsers in Selective mode
+
+Starting with the version 2.4, when switching to Selective mode, all browsers on the system are automatically added to the exceptions list for more convenience. This change applies under the following conditions:
```

Comment:
AdGuard VPN for iOS is the only VPN product where there still are Regular and Selective mode. In other products, both apps and extensions, it's called differently. Please check that before writing